gtk/src/toga_gtk/dialogs.py

```diff
@@ -6,7 +6,7 @@
 
 class BaseDialog:
     def __init__(self):
-        loop = asyncio.get_event_loop()
+        loop = asyncio.new_event_loop()
```

Comment:
This one was correct - at the point you're opening a dialog, there *must* be an event loop; if an event loop *doesn't* exist, the warning/error raised by asyncio is correct. 

```suggestion
        loop = asyncio.get_event_loop()
```

Reply:
I made the changes